test/rules-google-fullstory.spec.ts

```diff
@@ -1,448 +0,0 @@
-import { expect } from 'chai';
```

Comment:
I split tests for the 2 Google rulesets (Enhanced Ecommerce and Event Measurement) into separate files. These are separate integrations/config options in FullStory and are captured as distinct rulesets in the DLO repo, so it felt natural to break them apart into separate files.